Dockerfile

```diff
@@ -1,51 +1,15 @@
-FROM ubuntu:22.10
+FROM geonode/geonode-base:master-ubuntu-22.10
```

Comment:
@giohappy if you agree too I would change the suffix into `latest-ubuntu-22.10` which is a bit more coherent with the docker nomenclature

Reply:
I agree. I will change the tag and republish the image